unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/AvatarFactory.cs

```diff
@@ -0,0 +1,6 @@
+public class AvatarFactory
```

Comment:
I can't find any reference to this class

Reply:
forgot to delete it sorry